lib/release_notes_bot/persists.ex

```diff
@@ -97,6 +143,23 @@ defmodule ReleaseNotesBot.Persists do
     end
   end
 
+  def select_provider_to_update(persistence_provider_id, sanitizer_changes) do
+    case persistence_provider_id do
+      # Confluence
+      1 ->
```

Comment:
hmmm, it might make sense to add a provider type to this table. Maybe an enum? Probably not important until we have more then one provider type to update.

Reply:
Would an Enum for this be acceptable as a variable scoped to this file `persists.ex`.
Something like: `@provider_type %{:confluence => 1, google_docs => 2}`. Then as we add more providers, we can refactor down the road as needed.

Reply:
Enums work, sure. 

Maybe more importantly though, if we only have one, we don't need to worry about it yet. Might as well drop the whole case statement, and just assume its confluence all the time.

Hardcoding on `provider_id` doesn't seem like the most stable way of managing this relationship though. This is a sequence field?

Reply:
You're right its not. My thoughts of this were that Confluence would always be the first (and right now only) entry in persistence_providers. Thats where I get provider_id from. Its Confluence's id in persistence_providers.

Reply:
Having this case statement has actually helped track down a few bugs where the relation between a project and its persistence_provider (it should be always confluence) was not there.

Reply:
I'm not against having a case statement, but having it be on the id field seems like a not great long term plan

Reply:
You've got it. I'll make a ticket for a refac